ios/Capacitor/Capacitor/CAPPluginCall.swift

```diff
@@ -62,10 +62,12 @@ public typealias PluginEventListener = CAPPluginCall
         return self.options.index(forKey: key) != nil
     }
 
+    @available(*, deprecated, message: "Use `resolve()`")
```

Comment:
on Xcode the message appears as 
> 
```
'success' is deprecated: Use `resolve()`
```
So I think it would be better to use it like this 

```suggestion
    @available(*, deprecated, message: "Use 'resolve'")
```

so it appears as `'success' is deprecated: Use 'resolve'`

Reply:
If I use `renamed: "resolve"` then this is the message that appears and there's an option to fix automatically:

<img width="380" alt="image" src="https://user-images.githubusercontent.com/236501/89561532-7b974480-d7cd-11ea-9f89-85d4733cece4.png">